triplea_maps.yaml

```diff
@@ -1131,6 +1131,12 @@
     <br>WWIIv3 1942 - Combat-Move Before Purchasing  (by Talibush)
     <br>  1942 except where you do your combat movement before making purchases.
     <br>
+    <br>WWIIv3 1941 Move-Buy-Move (by Cernel)
+    <br>  A mod of WWIIv3 1941 where you can Combat Move both before and after purchasing units, using your normal total movement abilities in either, for easier gameplay.
+    <br>
+    <br>WWIIv3 1942 Move-Buy-Move (by Cernel)
+    <br>  A mod of WWIIv3 1942 where you can Combat Move both before and after purchasing units, using your normal total movement abilities in either, for easier gameplay.
+    <br>
```

Comment:
Duplicated? You mean with the similar previous mod description? This is 1942 and the previous one is 1941.